helm/hpcc/templates/_helpers.tpl

```diff
@@ -556,3 +556,20 @@ Generate list of available services
   public: {{ $esp.public }}
 {{ end -}}
 {{- end -}}
+
+
+{{/*
+Add resource object
+Pass in a dictionary with root and me defined
+*/}}
+{{- define "hpcc.addResources" }}
+{{- if .me.resources }}
+resources:
+{{- toYaml .me.resources | nindent 2 -}}
+{{ else }}
+{{- if .root.Values.global.resources }}
```

Comment:
Does it make sense to have global resources?  If so it would make more sense for them to be defined per component type.  Also, they should be merged with the local overrides, otherwise defining one limit locally may disable multiple global limits.

Reply:
I agree that would be a good scheme. 
A per component type set of defaults, used as a base, overridden by any per instance definitions, and/or any workunit overrides in the case of jobs (merged in both cases).